providers/core/core.unprivileged.js

```diff
@@ -185,6 +188,45 @@ Core_unprivileged.prototype.bindChannel = function(identifier, continuation, sou
 };
 
 /**
+ * Require a dynamic dependency for your freedom module.
+ * If new permissions are needed beyone what are already available to the
+ * freedom context, the user will need to approve of the requested permissions.
+ * @method require
+ * @param {String} manifest The URL of the manifest to require.
+ * @param {String} api The API of the dependency to expose if not default.
+ * @param {Function} callback The function to call with the dependency.
+ */
+Core_unprivileged.prototype.require = function (manifest, api, callback) {
+  if (this.manager.delegate && this.manager.toDelegate.core &&
```

Comment:
This is better abstracted out as a isInModule and reused in other functions as well. When our notion of what it means to be in a module changes, that makes it easier to change